tests/vxlan/test_vxlan_decap.py

```diff
@@ -190,4 +190,5 @@ def test_vxlan_decap(setup, vxlan_status, duthosts, rand_one_dut_hostname,
                        "dut_hostname": duthost.host.options[
                            'inventory_manager'].get_host(duthost.hostname).vars['ansible_host']},
                qlen=10000,
-               log_file=log_file)
+               log_file=log_file,
+               is_python3=True)
```

Comment:
why not change api signature of "ptf_runner" to make "is_python3" equal true by default.